### DISCUSSION TITLE
Shadow

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#525] [General] [answered]
Hey folks, do we have a global shadow pattern for images? I've got images from Brand team for Functions Overview page and it has white color background sticking out therefore when I attach it to the Overview page it blends with page background. 

here is the image: https://drive.google.com/file/d/1WmJyeFB9UwQlIBVkmsHargkl0GqYNZVP/view?usp=sharing

**Comment by richbachman** [comment, marked as answer]
Hi @sang-yeo. Right now we would defer to the brand designer on how to handle the shadow in this image. Avoiding shadows in imagery is preferred for now, so we don't end up clashing with future suggestions. One idea is to change the white part of the image to be a light gray.

We may have more guidance when UDL is done.

**Reply by kate-porter** [reply]
You could also make the entire circle green with just a white checkmark inside it.

**Comment by richbachman** [comment]
Hi @sang-yeo. Just checking in to see if you needed any more direction on this. Thanks.

**Reply by sang-yeo** [reply]
Hey @richbachman All good here. I requested different background color to our brand team. Thanks for your suggesting!
